### ISSUE TITLE
"Click" test helper does not work

### ISSUE THREAD

**Issue by arkadiyk** [runspired/ember-mobiletouch#49]
I managed to make it work by replacing 

`run['default']($el, 'click');`  with `run['default']($el, $el.trigger, 'tap');`

in `ember/ember-testing.js` `click` method. 
I guess it should be overwritten by the addon.

**Comment by kbullaughey** [comment]
An alternative is to use the `triggerEvent` helper to send a `tap` event:

    triggerEvent('.some .selector', 'tap');

**Comment by runspired** [comment]
I'd prefer to provide explicit gesture helpers to overwriting the `click` helper.  Once the full version of `ember 2.0` actions and events lands (looking like that will be around ember 2.1 right now), overriding `click` will likely no longer be necessary.

The solution to this currently is definitely to use the `triggerEvent` helper instead of `click`.

**Comment by runspired** [comment]
@kbullaughey you in the ember slack yet? I've got a group in there for dev talk on this addon.

**Comment by kbullaughey** [comment]
I am not. Didn't know ember folk were using Slack. I just joined the ember-mobile channel started by you, is that the one? Seems like I can't go very far back in the archive. I started poking around the ember-mobiletouch 2.0 beta and it seemed in too much flux for me to update to at this point for my main project. But I would be interested in having a better sense of what the future direction of ember-mobiletouch will be. Is that written down anywhere?
